runtime/moonriver/tests/integration_test.rs

```diff
@@ -292,7 +291,6 @@ fn verify_pallet_indices() {
 	is_pallet_index::<moonriver_runtime::EthereumChainId>(50);
 	is_pallet_index::<moonriver_runtime::EVM>(51);
 	is_pallet_index::<moonriver_runtime::Ethereum>(52);
-	is_pallet_index::<moonriver_runtime::BaseFee>(53);
```

Comment:
It might be worth leaving a comment here to remind us not to use this pallet index again. I'm assuming we shouldn't, but maybe there's no good reason to do that. In any case, I think a comment is harmless.

If we want to be more strict about it, we could add a test case that shows there is nothing at this pallet index.